src/autoscaler/configutil/cf.go

```diff
@@ -27,12 +27,12 @@ type VCAPConfiguration struct {
 	appEnv *cfenv.App
 }
 
-func NewVCAPConfigurationReader() (*VCAPConfiguration, error) {
+func NewVCAPConfigurationReader() *VCAPConfiguration {
 	appEnv, err := cfenv.Current()
 	if err != nil {
-		fmt.Println("failed to read VCAP_APPLICATION environment variable")
+		fmt.Printf("failed to read VCAP environment variables: %s\n", err)
```

Comment:
Can we use logger here ?

Reply:
Eventually I would like that, but we initialize the logger after creating the config instance. This configutil is used when creating the api config. So It requires a further refactor. I would add a issue for now on this ?

Reply:
PR is coming to for the improvement

Reply:
Ok a different solution for this here 32fa02be8, see commit description.

Reply:
Ok a different solution for this here 32fa02, see commit description.